pkg/cluster/cluster.go

```diff
@@ -138,6 +138,16 @@ func (c *Cluster) prepareBackupAndRestore() error {
 		}
 	}
 
+	// We use backup RS to mark that we have done backup creation and even restore process before.
```

Comment:
move this out of this func? create a new func called IsBackupPrepared. Call that fun before we call prepareBackupAndRestore? I feel it would be easier to read.